src/serialization.cpp

```diff
@@ -158,58 +158,34 @@ static void serializeContent(std::string* content, ov::Tensor& tensor) {
     }
 }
 
+#define serializeByDatatype(contents, datatype)                                            \
```

Comment:
By convention use capital letters to define macro's
-> #define SERIALIZE_BY_DATATYPE(CONTENTS, DATATYPE)